openwisp_controller/config/api/views.py

```diff
@@ -110,6 +114,17 @@ def retrieve(self, request, *args, **kwargs):
         return BaseConfigAdmin.download_view(self, request, pk=kwargs['pk'])
 
 
+class DeviceGroupListCreateView(ProtectedAPIMixin, ListCreateAPIView):
+    serializer_class = DeviceGroupSerializer
+    queryset = DeviceGroup.objects.select_related('organization').order_by('-created')
+    pagination_class = ListViewPagination
+
+
+class DeviceGroupDetailView(ProtectedAPIMixin, ListCreateAPIView):
```

Comment:
here you're inherting `ListCreateAPIView`.

Please spend a bit more time testing this.

Reply:
I caught this on in #500. Forgot to push changes on this PR.